### DISCUSSION TITLE
Space & Collection overflow menus default propogation(?) issue

### DISCUSSION THREAD

**Discussion by MitchellPeck** [GitbookIO/community#638] [0. General bugs and questions]
When viewing a space in the center window with sub-spaces or collections, there is an overflow menu. However, when clicking on the overflow, it also opens the space/collection.

**Comment by aikoii** [comment]
@MitchellPeck thanks for flagging this! It does seem like a bug 👷‍♀️👀

**Comment by aikoii** [comment]
@MitchellPeck This should be fixed 🎉 Also, you'll notice a small change: we moved the option to open the collection page in the collection's options palette:

![CleanShot 2024-06-14 at 14 43 08@2x](https://github.com/GitbookIO/community/assets/106068994/c140e74e-c950-4c7a-81bb-0210a75caf86)
